### PR TITLE
Math magicians: fetch data from API

### PR DESCRIPTION
**Pull request by HossainAraf**
Method Implemented: 

- Use the following external API: [API Ninjas](https://api-ninjas.com/api/quotes);
- Create a new component for displaying a quote;
- Fetch data inside of a new component;
- Show a loading state;
- Show an error state;
- Import the new component into your main component (next to the Calculator) & It should be on the same page as the Calculator
- Update style

### CODE REVIEW

File: src/components/DisplayQuote.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        const response = await fetch(apiURL, options);
+        const data = await response.json();
+        setJoke(data[0].joke);
+      } catch (error) {
+        setError(true);
+      }
+      setLoading(false);
+    };
+
+    fetchData(); // Call the function to fetch data when the component mounts
```
Comment (DeliceLydia) [inline src/components/DisplayQuote.js]: - [ ] I suggest that you change a function to `Quote` not `Joke` as the requirement says
- [ ] Please change the API from `Joke` to `Quote` you can refer to this [link](https://api-ninjas.com/api/quotes)

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
### HI @HossainAraf, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The `useEffect` hook has been used correctly
- There is a `loading` state
- There is an `error` state


## Required Changes ♻️
_Check the comments under the review._



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/DisplayQuote.js
Comment (Alusp) [inline src/components/DisplayQuote.js:1]: - [Optional]  When I refresh the browser two quotes display one after the other, it is supposed to be one quote per refresh. Please resolve that.

**Review by Alusp — APPROVED** [review]
Hi @HossainAraf, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congrats

## Highlights

- [ ] The useEffect hook has been used correctly :clap: 
- [ ] There is a loading state :clap: 
- [ ] There is an error state :clap: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**